cluster-autoscaler/integration/logs/autoscaler_processs.log

```diff
@@ -0,0 +1,4061 @@
+I0708 14:05:09.210791   60673 main.go:647] Cluster Autoscaler 1.30.1
```

Comment:
This file is added by error I guess. Can you please remove it?

Reply:
done